cli/azd/pkg/tools/javac/javac_test.go

```diff
@@ -21,7 +21,7 @@ func TestCheckInstalledVersion(t *testing.T) {
 	require.NoError(t, os.Mkdir(javaHomeBin, 0755))
 
 	placeJavac(t, javaHomeBin)
-	ostest.Setenv(t, "JAVA_HOME", javaHome)
```

Comment:
I was hoping to introduce consistency by having `<package>test` for test functionality surrounding golang's std package(s). And since we needed to introduce some helpers from unsetting env amongst other things, that it'd make sense to treat that as the facade, where `ostest.Setenv` could simply call `t.Setenv`. I can see using what's out there could be more idiomatic, but I almost like the consistency better.

Reply:
Hmm.. I'm learning more towards the facade when we have cases like:
```golang
ostest.Unsetenv(t, azdo.AzDoPatName)
t.Setenv(azdo.AzDoEnvironmentOrgName, "testOrg")
```

definitely seems a little weird to have disjunct APIs for handling similar things.

Reply:
Yeah, I guess I come from the side of "Most gophers know what `t.Setenv` does and so using that directly instead of a wrapper called `ostest.Setenv` (where you have to go examine what it does) was a little more confusing.  But I agree there's a nice symmetry here.

I will introduce `ostest.Setenv` and just add a comment that says it is the same as calling `t.Setenv` directly.